dask/blockwise.py

```diff
@@ -183,6 +184,9 @@ def __init__(
         self.concatenate = concatenate
         self.new_axes = new_axes or {}
 
+    def __repr__(self):
+        return "Blockwise<{} -> {}>".format(self.indices, self.output)
```

Comment:
Thanks for this.  Also maybe 
```suggestion
        return "Blockwise<{} -> {}>".format(self.indices, self.output)
        
    __str__ = __repr__
```

Or does Python do this automatically?